src/llm-error-pages/guidance-handler.js

```diff
@@ -59,14 +59,35 @@ export default async function handler(message, context) {
     const sheet = workbook.worksheets[0] || workbook.addWorksheet('data');
 
     const baseUrl = site.getBaseURL?.() || 'https://example.com';
+    const SPREADSHEET_COLUMNS = [
```

Comment:
given this is shared between handler and guidance-handler, can we move it to utils and used in both audits?